guides/8-perpetual-exchange/2-how-it-works.md

```diff
@@ -0,0 +1,346 @@
+
+#### Collateral Management
+
+Traders can deposit or withdraw collateral from the position to manage the position's margin.
+
+- When traders deposit collateral, the liquidation price and leverage for the short position increases as the maintenance margin increases.&#x20;
+- When traders withdraw collateral, the liquidation price and leverage for the short position decreases as the maintenance margin decreases.
+
+#### Underlying Collateral
+
+The underlying collateral for a short position is either one of the USDC or USDT stablecoin, depending on the stablecoins' **utilization rates** at the time when the position is opened:
+
+| Position   | Collateral  |
+| ---------- | ----------- |
+| Short SOL  | USDC / USDT |
+| Short wETH | USDC / USDT |
+| Short wBTC | USDC / USDT |
+
+Profits and collateral withdrawals are paid out to traders in the stablecoin used as the underlying collateral for the position.
+
+For example, a trader with a profitable short SOL position with USDC as the underlying collateral will receive USDC when they close the position or withdraw collateral.
+
+### Take-Profit / Stop-Loss Orders
+
+An active Associate Token Account (ATA) is needed for TP/SL to be triggered and executed:
+- ETH ATA for ETH-Long
+- wBTC ATA for wBTC-Long
+- USDC or USDT ATA for ALL Short positions
+
+An ATA will be automatically created for you when you create a TP/SL.
+
+Please note that the TP/SL will not be triggered if you close the respective ATA needed.
+
+In the event of **manually closing a position**, the associated trigger orders will be automatically canceled and closed, providing a seamless experience when opening new future positions.
+
+:::info
+While `TP/SL` orders are set, keepers will monitor the mark price, when reaching the specified price level, TP/SL will close the whole position. More info on keepers [here](#keepers).
+:::
+
+### PnL
+
+Profit and loss calculations directly corresponds to the size of your **position**. Here are two examples to illustrate this:
+
+1. Long position example:
+   * Position: 1,000 USD long on SOL
+   * If SOL price increases by 10%: You profit 100 USD
+   * If SOL price decreases by 10%: You lose 100 USD
+2. Short position example:
+   * Position: 1,000 USD short on SOL
+   * If SOL price decreases by 10%: You profit 100 USD
+   * If SOL price increases by 10%: You lose 100 USD
+
+In both cases, the profit or loss is 10% of the **position size**, matching the percentage change in SOL's price.
+
+:::info
+This example demonstrates a simplified calculation of PnL without factoring in the fees associated with Jupiter Perps. More info on fees [here](#fees).
+:::
+
+## Managing Leverage
+
+### Leverage
+
+To allow for leverage, traders borrow assets from the pool to create a larger position. To create a 2x long position SOL-USD, the other 1x SOL will be borrowed from the pool.
```

Comment:
the borrow rate is calculate according to the size of position only, without taking collateral into consideration